main/test/ca/uwaterloo/flix/library/TestInputStream.flix

```diff
@@ -0,0 +1,80 @@
+use Core/Io/InputStream.{available, close, fromBuffer, InputStream, readByte, readBuffer, readBufferWithLength};
+use Core/Io/InputStream.InputStream.InputStream;
+
+namespace TestInputStream {
+    def testBuffer(): InputStream & Impure = fromBuffer([0 as Int8, 1 as Int8, 2 as Int8])
+
+    @test
+    def fromBuffer(): InputStream & Impure = fromBuffer([0 as Int8, 1 as Int8, 2 as Int8])
+
+    @test
+    def available(): Bool & Impure = {
+        let r = available(testBuffer()) |> Result.map(a -> a == 3) ;
+        Result.getWithDefault(r, false)
+    }
+
+    @test
+    def readByte01(): Bool & Impure = {
```

Comment:
Good to see that you started writing some tests. In general we aim for 3-5 tests for each (public) function.